scripts/parseyaml.py

```diff
@@ -22,7 +22,7 @@ def _diff(a,b):
 			if k == 'xfer_size':
 				d[k] = v2[k]
 			elif k == 'Gb/sec' or k == 'usec/xfer':
-				d[k] = v2[k] - v1[k]
+				d[k] = float((v2[k] - v1[k])) / float(v2[k]) * 100
```

Comment:
Shouldn't this be `float((v2[k] - v1[k])) / float(v1[k]) * 100`? I'm assuming v1 has the old perf numbers and v2 has the new.